Documentation/networking.md

```diff
@@ -91,11 +104,15 @@ It creates a virtual ethernet pair (akin to a pipe) and places one end into pod
 
 #### bridge
 
-Like the ptp type, `bridge` will also create a veth pair and place one end into the pod. However the host end of the veth will be plugged into a linux-bridge.
+Like the ptp type, `bridge` will also create a veth pair and place one end into the pod.
```

Comment:
Like the ptp type, `bridge` will create a veth pair and attach one end to the pod.